docs/ISSUE_TEMPLATE.md

```diff
@@ -19,7 +20,7 @@ Please provide an ordered list on how to reproduce the issue.
 
 **Steps to Reproduce**:
 
-if relevant, please include a screenshot.
+if relevant, please include a screenshot, logs.
```

Comment:
if relevant, please include screenshots and logs.

Reply:
I will change 👍